cmd/buildah/common.go

```diff
@@ -108,3 +111,50 @@ func getDateAndDigestAndSize(image storage.Image, store storage.Store) (time.Tim
 	}
 	return created, manifestDigest, imgSize, err
 }
+
+// systemContextFromGlobalOptions returns a SystemContext populated with values
+// per the input parameters provided by the caller for the use in authentication.
+func systemContextFromGlobalOptions(c *cli.Context) (*types.SystemContext, error) {
```

Comment:
I'm don't think so, they're passed  down through the cli.  I stole this from skopeo fwiw.